stdlib/2and3/grp.pyi

```diff
@@ -0,0 +1,10 @@
+from typing import List, NamedTuple
+
+struct_group = NamedTuple("struct_group", [("gr_name", str),
+                                           ("gr_passwd", str),
+                                           ("gr_gid", int),
+                                           ("gr_mem", List[str])])
```

Comment:
Right, OK. All I tried was

```
import grp
grp.struct_group((None, None, None, None))
```

and that worked, but you're not really supposed to create `struct_group` by yourself, so more constrained types are the right thing to do.